Client/Frontend/Reader/ReaderMode.js

```diff
@@ -119,6 +119,23 @@ var _firefox_ReaderMode = {
             }
         }
     },
+
+    // Loop over the style sheets, find all the the @font-face declarations and fix the
+    // relative src attributes so that they point to our embedded web server.
+    fixFontFaces: function(webServerBase) {
```

Comment:
Can't we use <code>startWithPort</code> like we do <a href="https://github.com/mozilla/firefox-ios/blob/07d93ca582c0c840a3d8edac6606e2c3f1dda234/ClientTests/WebServerTests.swift#L19">here</a>?